app/assets/javascripts/tribo_viva/checkouts/checkout.js.coffee

```diff
@@ -0,0 +1,97 @@
+TriboViva.Checkouts ?= {}
+
+TriboViva.Checkouts.Checkout =
+  init: ->
+    Iugu.setAccountID("79d7660c-6169-4e07-aed9-8a0883d5249a")
+    Iugu.setTestMode(true) # disable on production
+
+    # Set mask for card expiration date
+    $(".js-credit-card-number").mask("9999 9999 9999 9999")
```

Comment:
não tem gem, só o js. e tem o https://github.com/firstopinion/formatter.js/blob/master/dist/jquery.formatter.js e o https://github.com/firstopinion/formatter.js/blob/master/dist/formatter.js no caso do iugu é o sem jquery. se for usar no app com jquery teria que baixar os dois (mas não tem pq)